File: php/Dランク早解きセット/【早解17】騒音レベル.php
<?php
    // 自分の得意な言語で
    // Let's チャレンジ！！

    $value = trim(fgets(STDIN)); // 入力値
    
    // 条件に合わせた結果の出力
    if($value < 30 ){
        echo 'quiet';
    }elseif($value < 50){
        echo 'normal';
    }elseif($value < 70){
        echo 'noisy';
    }elseif(70 <= $value){
        echo 'very noisy';
    }else{
    }
?>

File: python/Dランク早解きセット/【早解17】騒音レベル.py
# coding: utf-8
# 自分の得意な言語で
# Let's チャレンジ！！
num = int(input())

if num < 30:
    print('quiet')
elif 30 <= num and num < 50:
    print('normal')
elif 50 <= num and num < 70:
    print('noisy')
else:
    print('very noisy')

